content/en/docs/reference/kubectl/cheatsheet.md

```diff
@@ -56,11 +56,12 @@ kubectl config view
 # get the password for the e2e user
 kubectl config view -o jsonpath='{.users[?(@.name == "e2e")].user.password}'
 
-kubectl config view -o jsonpath='{.users[].name}'    # display the first user
-kubectl config view -o jsonpath='{.users[*].name}'   # get a list of users
-kubectl config get-contexts                          # display list of contexts 
-kubectl config current-context                       # display the current-context
-kubectl config use-context my-cluster-name           # set the default context to my-cluster-name
+kubectl config view -o jsonpath='{.users[].name}'                                            # display the first user
+kubectl config view -o jsonpath='{.users[*].name}'                                           # get a list of users
+kubectl config view --raw -o jsonpath="{.users[2].user.client-certificate-data}" | base64 -d # display certifications
```

Comment:
Using a Go template would be more appropriate here, I think, because Go templates can base64 decode.

Why subscript 2? That's not obvious.